test/integration/test_dmap.py

```diff
@@ -0,0 +1,302 @@
+import pydarn
+from pydarn import DmapScalar, DmapArray
+import unittest
+import numpy as np
+import logging
+import bz2
+import os
+
+import rawacf_data_sets
+import dmap_data_sets
+import copy
+from collections import OrderedDict
+
+# Test files
+rawacf_stream = "../testfiles/20170410.1801.00.sas.stream.rawacf.bz2"
```

Comment:
*comment here applies to other files too

Reply:
Maybe but I was also thinking of linking this to FRDR publication to keep files consistent otherwise testing values in the later tests or finding corrupt files may be challenging for a user to do thus my test would fail. I think unit tests should be self-contained so that you can actually predict and test behaviors for white box testing. 

Note: we don't include superdarn test files in this GitHub.

Reply:
Alright, lets just add a TODO for clean up of this then. The current state does make it difficult for anyone to checkout the code and run the tests.